src/__tests__/actions/editArticle/__test__/editArticle.test.js

```diff
@@ -31,7 +32,7 @@ describe("Edit an article test", () => {
     store.clearActions();
   });
 
-  it("tests that we should be able to update an article", () => {
+  it("tests update an article", () => {
```

Comment:
Hey @BagzieGracious, we can make this more descriptive, something like `it('should update an article)`

Reply:
I have also fixed that by giving a descriptive message